.github/workflows/test.yml

```diff
@@ -6,7 +6,7 @@ env:
   TEST_RESULTS: /tmp/test-results # path to where test results are saved
   CONSUL_VERSION: 1.13.0-alpha1 # Consul's OSS version to use in tests
   CONSUL_ENT_VERSION: 1.13.0-alpha1+ent # Consul's enterprise version to use in tests
-  GOTESTSUM_VERSION: 1.6.4 # You cannot use environment variables with workflows. The gotestsum version is hardcoded in the reusable workflows too.
+  GOTESTSUM_VERSION: 1.8.1 # You cannot use environment variables with workflows. The gotestsum version is hardcoded in the reusable workflows too.
 
 jobs:
   get-go-version:
```

Comment:
I think we also need to update `.goversion` file as that's what's used by the `test` GHA pipeline

Reply:
👍 `.go-version` is updated.